src/containers/quiz/quiz-info/QuizInfo.tsx

```diff
@@ -12,9 +12,18 @@ import styles from '~/containers/quiz/quiz-info/QuizInfo.styles'
 import {
   getFormattedDate,
   formatTime,
-  formatTimeDifference
+  formatTimeDifference,
+  spliceSx
 } from '~/utils/helper-functions'
 
+import DividerComponent from '~/design-system/components/divider/Divider'
+import { FinishedAttempts, QuizAttempt, QuizTimeLimit } from '~/types'
+import { getQuizTimeLimitFields } from '~/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.constants'
+import { TFunction } from 'i18next'
+import TimeLimitReminder from '~/containers/quiz/time-limit-reminder/TimeLimitReminder'
+import { useState } from 'react'
+import { Alert } from '@mui/material'
```

Comment:
```suggestion
import { Alert } from 'path-to-our-design-system'
```
We have `Alert` in our design system, does something wrong with it?

Reply:
Yes, there was some problems with clear icon, because in Quiz Preview in the component of our design system, we do not need the functionality of the clear icon. Therefore, the component from the MUI was used to use my alert.